### DISCUSSION TITLE
Who directed The Godfather films?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#186] [Galaxy Trivia] [answered]
**Hint:** FFC

**Comment by thenithinbalaji** [comment]
Francis Ford Coppola

**Comment by thuongtruong1009** [comment]
The first two Godfather films were directed by Francis Ford Coppola.

**Comment by muink** [comment, marked as answer]
Francis Ford Coppola

**Reply by ParanoidUser** [reply]
You got it! One more to go for the `Galaxy Brain` badge. 😊
